### DISCUSSION TITLE
DNS records implementation

### DISCUSSION THREAD

**Discussion by onboardng** [vercel/community#1232] [Help] [answered]
### Summary

i saw the guide on how to update my dns record from my domain provider so my app can point to my domain... but the guide seems impossible.. for example.. i want my app to point to 'schools.onboard.com.ng' so vercvel told me to enter the following in my dns records: 
type: CNAME,
 name: schools, 
value: cname.vercel-dns.com.

unfortunately, my dns provider said the name: schools is invalid, i  have to enter something valid like "schools.onboard.com.ng" 
which i also did but my app is still not pointing to that url.. I would appreciate some swift look and solution to this..

thanks

**Comment by amyegan** [comment, marked as answer]
The exact configuration depends on your domain provider. If you share the name of your provider, I can help you find their documentation to figure out the right values for your subdomain.
